samples/Http2SampleApp/Program.cs

```diff
@@ -24,7 +24,8 @@ public static void Main(string[] args)
                     factory.SetMinimumLevel(LogLevel.Trace);
                     factory.AddConsole();
                 })
-                .UseKestrel((context, options) =>
+                .UseKestrel()
+                .ConfigureKestrel((context, options) =>
```

Comment:
Why not change the other samples?

Reply:
In these samples it doesn't matter which you use, I only changed this one as a proof of concept.